test/CMakeLists.txt

```diff
@@ -63,7 +63,10 @@ add_subdirectory(files)
 add_subdirectory(util)
 
 if(NGRAPH_CPU_ENABLE)
-    set(SRC ${SRC} backend_performance.cpp codegen.cpp cpu_fusion.cpp cpu_test.cpp)
+    set(SRC ${SRC} backend_performance.cpp cpu_fusion.cpp cpu_test.cpp)
```

Comment:
I'm all for just removing the file backend_performance.cpp from the test directory.